### PR TITLE
Adding MARS

### PR DESCRIPTION
**Pull request by volcano1998**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by volcano1998** [comment]
@BiocondaBot please add label

**Comment by volcano1998** [comment]
@BiocondaBot please add label

File: recipes/mars/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+test:
+  commands:
+    - MARS_step1 --help
+    - MARS_step2 --help
+
+
+about:
+  home: https://github.com/maiziex/MARS
+  license: MIT
+  summary: Multiple Alignment-based Refinement of SVs(MARS)
```
Comment (simonbray) [inline recipes/mars/meta.yaml:40]: Can you add `license_file` here as well please?

Could you also add yourself as a maintainer to this recipe?

Comment (volcano1998) [reply recipes/mars/meta.yaml:40]: should I create a new pull request? Or how can I edit it based on this pr?

Comment (simonbray) [reply recipes/mars/meta.yaml:40]: No, please don't create a new PR. You can just push new commits to your branch.

Or if you aren't familiar with git you should be able to edit here: https://github.com/volcano1998/bioconda-recipes/edit/yourpackage3/recipes/mars/meta.yaml

Comment (volcano1998) [reply recipes/mars/meta.yaml:41]: Hello Simon, I already added license file, thank you for your direction!  However, I'm not sure about "adding myself as a maintainer". Could you give me some guide if you don't mind? Thank you!

Comment (simonbray) [reply recipes/mars/meta.yaml]: Hi @volcano1998, I added it here: https://github.com/bioconda/bioconda-recipes/pull/30685/commits/3b865eb3f04fa4ee8e32d998bd5abca56fba7000

**Review by simonbray — APPROVED** [review]

**Comment by simonbray** [comment]
@bioconda-bot please merge

**Pull request merged**